homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -35,6 +51,41 @@
 
 SUCCESS = ["ok"]
 
+SensorType = namedtuple("SensorType", ["unit", "icon", "device_class"])
+GATEWAY_SENSOR_TYPES = {
+    "temperature": SensorType(
+        unit=TEMP_CELSIUS, icon=None, device_class=DEVICE_CLASS_TEMPERATURE
+    ),
+    "humidity": SensorType(
+        unit=UNIT_PERCENTAGE, icon=None, device_class=DEVICE_CLASS_HUMIDITY
+    ),
+    "pressure": SensorType(unit=PRESSURE_HPA, icon=None, device_class=DEVICE_CLASS_PRESSURE),
+}
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up the Xiaomi sensor from a config entry."""
+    entities = []
+
+    # Gateway subdevices
+    if config_entry.data[CONF_FLOW_TYPE] == CONF_GATEWAY:
+        gateway = hass.data[DOMAIN][config_entry.entry_id]
+        subdevices = gateway.devices
+        for subdevice in subdevices:
+            if (
+                subdevice.type == DeviceType.AqaraHT
+                or subdevice.type == DeviceType.SensorHT
+            ):
+                sensor_variables = ["temperature", "humidity", "pressure"]
+                entities.extend(
+                    [
+                        XiaomiGatewaySensor(subdevice, variable)
+                        for variable in sensor_variables
+                    ]
+                )
```

Comment:
Please separate words with underscore.
```suggestion
        sub_devices = gateway.devices
        for sub_device in sub_devices:
            if (
                sub_device.type == DeviceType.AqaraHT
                or sub_device.type == DeviceType.SensorHT
            ):
                sensor_variables = ["temperature", "humidity", "pressure"]
                entities.extend(
                    [
                        XiaomiGatewaySensor(sub_device, variable)
                        for variable in sensor_variables
                    ]
                )
```